place/services/maps_mcp.py

```
@@ -0,0 +1,39 @@
+    payload = {
+        "query": keyword,
+        "location": {
+            "latitude": latitude,
+            "longitude": longitude
+        },
+        "radius": radius
+    }
+
+    try:
+        async with streamablehttp_client(url) as (read_stream, write_stream, _):
+            async with ClientSession(read_stream, write_stream) as session:
```

Ruff (0.12.2)

Use a single `with` statement with multiple contexts instead of nested `with` statements

(SIM117)